magento/pwa-studio
# Adding information about new module

## Description

Added documentation about the new `venia-data-collector` as part of the PREX integration.

## Acceptance

### Verification Stakeholders

### Specification

## Verification Steps

1. Go to https://magento.github.io/pwa-studio/product-recs/#install-the-product-recommendations-module
2. Verify there is information about the `venia-data-collector`.
